core/plugins/org.fusesource.ide.camel.model.service.core/src/org/fusesource/ide/camel/model/service/core/model/AbstractCamelModelElement.java

```diff
@@ -1754,4 +1753,16 @@ public Element createElement(String nodeTypeId, String namespace) {
 	public String getTranslatedNodeName() {
 		return CamelUtils.getTranslatedNodeName(getXmlNode());
 	}
+	
+	/**
+	 * Notify listeners after remove Context Element
```

Comment:
it is not only about ContextElement, it is about removal of the modelElement which is passed in parameter